docs/content/en/docs/concepts/architecture/components/KLTcomponents.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Lifecycle Toolkit Components
+linktitle: KLT Components
+description: Basic understanding of Keptn Lifecycle Toolkit Components
+weight: 80
+cascade:
+---
+
+### Keptn Lifecycle Toolkit Components
+
+The Keptn Lifecycle Toolkit consists of two main components:
+
+* Keptn Lifecycle Operator, splits into two separate operators:
```

Comment:
```suggestion
* Keptn Lifecycle Operator, which splits into two separate operators
in Release 0.7.0 and later:
```
Since this is a big change, i like specifying that here.

Reply:
yes, I agree.